packages/application-shell/src/components/locale-switcher/locale-switcher.tsx

```diff
@@ -59,14 +62,53 @@ const CustomMenuList = (props: MenuListProps) => {
   return <components.MenuList {...props}>{props.children}</components.MenuList>;
 };
 
+export const CustomGroupHeading = (
+  props: GroupHeadingProps & { setIsOpen: (value: boolean) => void }
+) => {
+  const { setIsOpen, ...groupProps } = props;
+  return (
+    <>
+      <components.GroupHeading
+        {...groupProps}
+        css={css`
+          display: flex;
+          gap: ${designTokens.spacing10};
+        `}
+      >
+        {groupProps.children}
+        <IconButton
+          icon={<InformationIcon />}
+          label="Locales info"
+          size="small"
+          onClick={() => setIsOpen(true)}
+        />
+      </components.GroupHeading>
+    </>
+  );
+};
+CustomGroupHeading.displayName = 'CustomGroupHeading';
+
 const LocaleSwitcher = (props: Props) => {
   const { setProjectDataLocale } = props;
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
Perhaps we can be more meaningful here.

What about "isDescriptionDialogOpened"?

Reply:
> Perhaps we can be more meaningful here.
> 
> What about "isDescriptionDialogOpened"?

Thanks for the suggestion! https://github.com/commercetools/merchant-center-application-kit/pull/3403/commits/bfd856ffc8d6fc3d997dc7e7f9a817802e4c944c